exercises/split-second-stopwatch/canonical-data.json

```diff
@@ -0,0 +1,194 @@
+{
+  "exercise": "split-second-stopwatch",
+  "comments": ["Times are formatted as 'HH:mm:ss'."],
```

Comment:
This may not be clear in terms of "What do these letters mean".  I mean to say that it may be intuitive that it is hour, minutes, seconds, but why the change in case for hours?

It can be useful to show a time where the `HH` is put in place, such as 13:14:15, to show the difference from a 12 hour clock to a twenty four hour clock.  To others, it will be clear, but via experience rather than inherent knowledge.

Reply:
Fair point! I'll update